### PR TITLE
LPS-164025

### PR DESCRIPTION
**Pull request by alinedoleron**
Ticket: https://issues.liferay.com/browse/LPS-164025

### CODE REVIEW

**Comment by alinedoleron** [comment]
ci:test:relevant

**Comment by naraandrade** [comment]
Just started reviewing :)

**Review by naraandrade — CHANGES_REQUESTED** [review]
@alinedoleron There's an error occurring when I try to access the Notification Queue page. Apparently, you're importing the file `./NotificationQueueEntryStatusDataRenderer.scss` in some file, but this .scss file doesn't exist.

**Comment by alinedoleron** [comment]
ci:test:sf

**Comment by alinedoleron** [comment]
ci:test:relevant

File: modules/apps/notification/notification-web/src/main/resources/META-INF/resources/js/NotificationQueueEntryStatusDataRenderer.tsx [hunk 1]
```diff
@@ -12,8 +12,50 @@
...
+	const toArray = to.split(',');
+	const lastItem = toArray.length > 1 ? toArray.pop() : null;
+	const toTooltipText =
+		toArray.length > 1
+			? toArray.join(',') +
+			  ' ' +
+			  Liferay.Language.get('and').toLowerCase() +
+			  lastItem +
+			  '.'
+			: to;
```
Comment (danvb112) [inline modules/apps/notification/notification-web/src/main/resources/META-INF/resources/js/NotificationQueueEntryStatusDataRenderer.tsx:34]: Hey @alinedoleron, to make the code inside the component as clean as possible, we can create a function outside the component to do this, and on line `57` we just call the function passing the `value` as a parameter, like:

```tsx

const getToTooltipText = (value: string) => {
	const toArray = value.split(',');
	const lastItem = toArray.length > 1 ? toArray.pop() : null;

	return toArray.length > 1
		? toArray.join(',') +
				' ' +
				Liferay.Language.get('and').toLowerCase() +
				lastItem +
				'.'
		: value;
};
```

File: modules/apps/notification/notification-web/src/main/resources/META-INF/resources/js/NotificationQueueEntryStatusDataRenderer.tsx [hunk 2]
```diff
@@ -12,8 +12,50 @@
...
+	const toValue = to.length > 25 ? to.slice(0, 25).trim() + '...' : to;
```
Comment (danvb112) [inline modules/apps/notification/notification-web/src/main/resources/META-INF/resources/js/NotificationQueueEntryStatusDataRenderer.tsx:35]: Hey @alinedoleron, you can pass this ternary condition directly in where you call this const and delete it

**Review by danvb112 — CHANGES_REQUESTED** [review]
Hey @alinedoleron, Great job on this PR, I only have two suggestions on it. If you have any question just ask!  :smile:

File: modules/apps/notification/notification-web/src/main/resources/META-INF/resources/js/components/EditNotificationTemplate.tsx [hunk 3]
```diff
@@ -469,8 +469,12 @@ export default function EditNotificationTemplate({
 														},
 													})
 												}
```
Comment (alinedoleron) [inline modules/apps/notification/notification-web/src/main/resources/META-INF/resources/js/components/EditNotificationTemplate.tsx:471]: H @danvb112. I'm sending this code to make an adjustment in a previous PR that [Brian asked me to do](https://github.com/brianchandotcom/liferay-portal/pull/124638).
Thanks.

File: modules/apps/notification/notification-web/src/main/resources/META-INF/resources/js/components/EditNotificationTemplate.tsx [hunk 4]
```diff
@@ -469,8 +469,12 @@ export default function EditNotificationTemplate({
...
-												placeholder={Liferay.Language.get(
-													'use-terms-to-configure-recipients-for-this-notifications'
+												placeholder={Liferay.Util.sub(
+													Liferay.Language.get(
+														'use-terms-to-configure-recipients-for-this-notifications-x'
+													),
+													'[%ENTRY_CREATOR%], [%BUYER_NAME%]',
+													'.'
```
Comment (alinedoleron) [inline modules/apps/notification/notification-web/src/main/resources/META-INF/resources/js/components/EditNotificationTemplate.tsx:477]: Hi @danvb112. I'm sending this code to make an adjustment in a previous PR that [Brian asked me to do](https://github.com/brianchandotcom/liferay-portal/pull/124638).
Thanks.

**Comment by alinedoleron** [comment]
ci:test:sf

**Comment by alinedoleron** [comment]
ci:test:relevant

**Review by danvb112 — APPROVED** [review]
LGTM! :tada:

**Comment by danvb112** [comment]
ci:forward

**Pull request closed without merging**